src/main/java/org/cactoos/map/Synced.java

```diff
@@ -150,12 +150,14 @@ public Synced(final Map<X, Y> map,
      */
     public Synced(final Map<X, Y> map) {
         super(
-            new org.cactoos.scalar.Synced<>(
-                () -> {
-                    final Map<X, Y> temp = new ConcurrentHashMap<>(0);
-                    temp.putAll(map);
-                    return temp;
-                }
+            new MapOf<>(
+                new org.cactoos.scalar.Synced<>(
+                    () -> {
+                        final Map<X, Y> temp = new ConcurrentHashMap<>(0);
```

Comment:
@andreoss here please replace all of this by just wrapping the map into `Collections.synchronizedMap` as with `list.Synced`

Reply:
@victornoel Done.